src/webfaf/problems.py

```diff
@@ -533,7 +534,7 @@ def item(problem_id):
     # are selected which is not a good representation.)
     k = min(len(bt_hashes), 10)
     a = 0
-    d = len(bt_hashes)/float(k)
+    d = len(bt_hashes)//float(k)
```

Comment:
I guess this should stay / as well.

Reply:
You are right, it should stay ```/``` because the second operator is ```float```.